### ISSUE TITLE
Displayed patterns should be anchored if not regexps. 

### ISSUE THREAD

**Issue by GLundh** [AxisCommunications/goalie-url-shortener#24]
Typical use case:

User A: Hey, what's the long complex link for service xyz?
User B: Hmm.. I think it is in our goalie service.. Let me check.
User B finds the link by searching in Goalie. But only the target is clickable. What User B *really* wants is to easily share the goalie vanity url, to make it easier for User A to find the link in the future *and* to spread awareness about goalie.

**Comment by AntonFriberg** [comment]
So what you would want is an icon for "Copy to Clipboard" that inserts the entire vanity url to clipboard?

**Comment by GLundh** [comment]
That would be cool. Though making it an anchor link I fine too. I can right click -> copy if I want it in the clipboard.. 

Regexps patterns are not really vanity urls or linkable, so those should probably be just text (like today).

**Comment by AntonFriberg** [comment]
So something like this?

![image](https://user-images.githubusercontent.com/14290517/67564318-ff8ead00-f722-11e9-9cf1-94a9e7aa2bba.png)

**Comment by GLundh** [comment]
Exactly like that.

**Comment by AntonFriberg** [comment]
What about using something like this to create strings that match the regex? This would make anchor links possible for any pattern. https://fent.github.io/randexp.js/

**Comment by GLundh** [comment]
:D Awesome. But can you control the length on it? So we get decently short urls out of it?

**Comment by AntonFriberg** [comment]
Yes! The length could be exposed as a configurable environment variable.
https://github.com/fent/randexp.js#default-range
